corehq/apps/app_manager/tests/test_util.py

```diff
@@ -93,36 +94,56 @@ class SchemaTest(SimpleTestCase):
     def setUp(self):
         self.factory = AppFactory()
 
-    def test_get_casedb_schema_empty_app(self):
```

Comment:
Should this test be replaced with a test for a form that doesn't use case management?